### ISSUE TITLE
Accessibility: Notification Settings

### ISSUE THREAD

**Issue by jd-alexander** [wordpress-mobile/WordPress-Android#10736]
#### Accessibility Scanner Audit
<img src="https://user-images.githubusercontent.com/1509205/68187586-e1fbe780-ff5b-11e9-893b-5a8b5be28542.png" alt="screenshot_WordPress_ 0,490 1440,622 _4294967305_2019-11-04-19_47_31" width="300" />

- [x] **Main Area** - Your Sites- `org.wordpress.android:id/header_text` - Text contrast

The item's text contrast ratio is 1.98. This ratio is based on an estimated foreground color of #B2B2B2 and an estimated background color of #F6F7F7. Consider increasing this item's text contrast ratio to 3.00 or greater.

- [x] **Followed Sites - Notification Dialog** - 
<img src="https://user-images.githubusercontent.com/1509205/68187707-31daae80-ff5c-11e9-8523-48f5c87ac5b4.png" alt="screenshot_WordPress_2019-11-04-19_49_15" width="300" />

New posts - Description - Text contrast
`org.wordpress.android:id/notifications_new_posts_description`

The item's text contrast ratio is 4.20. This ratio is based on an estimated foreground color of `#787C82` and an estimated background color of `#FFFFFF`. Consider using a contrast ratio greater than 4.50 for small text, or 3.00 for large text.

- [x] **Your Sites - Notification Screen** 

* Header - Touch target
	
This item's height is 38dp. Consider making the height of this touch target 48dp or larger.
	
* Notification Setting Pop up Dialog -  org.wordpress.android:id/row_containerTouch target
	
This item's height is 44dp. Consider making the height of this touch target 48dp or larger.

- [ ] **Notification Search** 

<img src="https://user-images.githubusercontent.com/1509205/68188243-98ac9780-ff5d-11e9-83df-f3820eab9c58.png" alt="screenshot_WordPress_2019-11-04-19_50_59" width="300" />

* Search Text - `org.wordpress.android:id/search_src_text` - Text contrast
	
The item's text contrast ratio is 2.95. This ratio is based on an estimated foreground color of #80B0C4 and an estimated background color of #006088. Consider increasing this item's text contrast ratio to 3.00 or greater.
	
* Search Text - `org.wordpress.android:id/search_src_text` - Touch target
	
This item's height is 36dp. Consider making the height of this touch target 48dp or larger.

	
#### TalkBack Audit

* #### Main Area 

- [ ] Your Sites & Other Header gains focus and then say it's disabled. This doesn't provide any valuable information to the user. 
	
- [ ] Followed Sites Notifications Settings Dialog

<img src="https://user-images.githubusercontent.com/1509205/68187794-76fee080-ff5c-11e9-8c84-66a5a3baf9b5.png" alt="Screenshot_20191104-192508" width="300" />

* New Post and the content below it can be read together. 
* Make the radio buttons below the Email new posts setting more descriptive. 

* #### Notification Search 

- [ ] When there's no search result then the app should announce this state and do the same when the results a
